src/config.ts

```diff
@@ -0,0 +1,13 @@
+export default {
+	clientURL: "https://chat.fribyte.no",
```

Comment:
```suggestion
	clientURL: "https://chat.fribyte.no",
	get channelName() {
		return process.env.NODE_ENV === "production" ? "dugnad" : "bot-test";
	},
	get isProd() {
		return process.env.NODE_ENV === "production";
	},
	get token() {
		return process.env.TOKEN ?? throw new Exception("Token is not defined in environment. Add 'TOKEN=<YOUR TOKEN>' to the file '.env' in root folder.");
	}
```

I like to put all config stuff and env variables accessible in the config object. And then only use config all other places in the code.

Reply:
Me likey. 
Also never seen the get syntax before, js is one hell of a language